### PR TITLE
Add support for ACL SAVE/LOAD commands

### PR DESCRIPTION
**Pull request by dengliming**
Resolves [https://github.com/redis/jedis/issues/2169](https://github.com/redis/jedis/issues/2169)

### CODE REVIEW

File: src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java [hunk 1]
```diff
@@ -81,5 +81,7 @@ String migrate(String host, int port, int destinationDB, int timeout, MigratePar
 
   byte[] aclLog(byte[] options);
 
-  // TODO: Implements ACL LOAD/SAVE commands
+  byte[] aclLoadBinary();
+
+  byte[] aclSaveBinary();
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java:86]:
```suggestion
  String aclLoad();

  String aclSave();
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]
When the is just `OK`, it's okay to skip separate method with `Binary`.

File: src/main/java/redis/clients/jedis/Jedis.java [hunk 2]
```diff
@@ -3908,6 +3908,18 @@ public String aclGenPass() {
...
 
+  @Override
+  public String aclLoad() {
+    client.aclLoad();
+    return client.getStatusCodeReply();
+  }
+
+  @Override
+  public String aclSave() {
+    client.aclSave();
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/Jedis.java:3919]:
```suggestion
    checkIsInMultiOrPipeline();
    client.aclSave();
```

File: src/main/java/redis/clients/jedis/Jedis.java [hunk 3]
```diff
@@ -3908,6 +3908,18 @@ public String aclGenPass() {
     return client.getStatusCodeReply();
   }
 
+  @Override
+  public String aclLoad() {
+    client.aclLoad();
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/Jedis.java:3913]:
```suggestion
    checkIsInMultiOrPipeline();
    client.aclLoad();
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]
Add methods in `Commands.java`

**Comment by dengliming** [comment]
@sazzad16 Thanks.

**Comment by sazzad16** [comment]
@dengliming Please add methods in Commands.java

**Comment by dengliming** [comment]
@sazzad16 I'm a little confused. Exactly which methods should be placed in `Commands ` and which methods should be placed in `AdvancedJedisCommands`? Seems `ACL ` commands all in `AdvancedJedisCommands`. such as `aclGenPass`. Can you explain it in detail? Thanks. : )

**Comment by sazzad16** [comment]
@dengliming Frankly, I'm not sure about this specific case. Historically, methods that can be used from Client.java are put in Commands.java. I'm just continuing to maintain it until we decide to drop it future.

**Comment by sazzad16** [comment]
@dengliming Hmm, it seems all the ACL methods are out of sync. Okay, never mind.

**Review by sazzad16 — APPROVED** [review]

**Comment by dengliming** [comment]
@sazzad16 Got it. Thanks for helping. : )

**Pull request merged**